locales/es_ES/pcsx2_Main.po

```diff
@@ -8,8 +8,8 @@ msgstr ""
 "Project-Id-Version: PCSX2 1.2.1\n"
```

Comment:
We can also update the project version to 1.5.0.

Reply:
Done 👍